src/main/java/billgates/database/MySQLDatabaseGateway.java

```diff
@@ -699,7 +699,7 @@ public void createUsersTable() {
 
 
             String query = """
-                    CREATE TABLE users
+                    CREATE TABLE IF NOT EXISTS users
```

Comment:
If we decide to do this, then maybe I will have to fix some of the tests as I manually tested for the "IF NOT EXISTS" part.